frontend/src/libs/components/modal/steps/delete-account/delete-acount-form.tsx

```diff
@@ -0,0 +1,113 @@
+import { type ChangeEvent, type FC } from 'react';
+
+import { useCallback, useState } from '#libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  onNext?: () => void;
+  onClose?: () => void;
+};
+
+const DeleteAccountForm: FC<Properties> = ({ onNext, onClose }) => {
+  const [isChecked, setIsChecked] = useState({
+    checkbox1: false,
+    checkbox2: false,
+    checkbox3: false,
+    checkbox4: false,
+  });
+
+  const [text, setText] = useState('');
+
+  const handleCheckboxChange = useCallback(
+    (event: ChangeEvent<HTMLInputElement>): void => {
+      const { name, checked } = event.target;
+      setIsChecked((previousState) => {
+        return { ...previousState, [name]: checked };
+      });
+    },
+    [],
+  );
+
+  const handleTextChange = useCallback(
+    (event: ChangeEvent<HTMLInputElement>): void => {
+      setText(event.target.value);
+    },
+    [],
+  );
+
+  const isNextDisabled = !Object.values(isChecked).some(Boolean);
+
+  return (
+    <div className={styles['modal-body']}>
+      <form className={styles['form']}>
+        <div>
+          <label>
+            <input
+              type="checkbox"
+              name="checkbox1"
+              onChange={handleCheckboxChange}
+              checked={isChecked.checkbox1}
+            />
+            I&apos;ve chose another app
```

Comment:
The same, we have common checkbox components, just create another preseted style there